docs/05.PORT-API.md

```diff
@@ -0,0 +1,210 @@
+# Reference
+
+## Termination
+
+It is questionable whether a library should be able to terminate an application. Any API function can signal an error (ex.: cannot allocate memory), so the engine use the termination approach with this port function.
```

Comment:
I would remove the "It is questionable whether a library should be able to terminate an application. " because the next sentence has no logical relation to it.